CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Relicensed C-PAC from BSD-3-Clause to LGPL-3.0-or-later
 
 ### Fixed
+- Fixed [bug](https://github.com/FCP-INDI/C-PAC/issues/1795) that was causing `cpac run` to fail when passing a manual random seed via `--random_seed`.
```

Comment:
Thanks for adding this. Can you please move the line up to the ["Unreleased" section](https://github.com/FCP-INDI/C-PAC/blob/b5c742a12e42b68ff78e2f039a24cdeb714216e1/CHANGELOG.md?plain=1#L17-L34)?

Reply:
whoops! good catch, sorry should be fixed now